jdaviz/configs/mosviz/tests/test_parsers.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import tempfile
+from zipfile import ZipFile
+import pathlib
+
+from astropy.utils.data import download_file
+
+from jdaviz.configs.mosviz.helper import MosViz
+
+@pytest.fixture
```

Comment:
This is unnecessary, use this:

https://github.com/spacetelescope/jdaviz/blob/519aaec9cc86b142e51d1b7f120ff70dd342124b/jdaviz/conftest.py#L28-L30